### PR TITLE
Dev workflow

### LINKED ISSUE #430: Dev workflow is currently very painful
image

### PR DESCRIPTION
**Pull request by Vibaswan**
Fixes #430

### CODE REVIEW

File: do.py [hunk 1]
```diff
@@ -459,6 +460,42 @@ def getstatusoutput(command):
     )
 
 
+def build(clean_env="false", sdk="all"):
+    print("\nStep 1: Set up virtaul env, if does not exsists already")
```
Comment (ashutshkumr) [inline do.py:464]: remove `if does not *` since you're already printing the messages below

File: do.py [hunk 2]
```diff
@@ -459,6 +460,42 @@ def getstatusoutput(command):
     )
 
 
+def build(clean_env="false", sdk="all"):
```
Comment (ashutshkumr) [inline do.py:463]: sdk should be the first argument as that's more frequently going to be used.

clean_env accepting false/true is not intuitive; maybe use `env_setup` with possible values as None and "clean"; we might add more later (e.g. reinstall_deps, etc.)

File: do.py (hunk 1 above)
Comment (Vibaswan) [reply do.py]: done

File: do.py (hunk 2 above)
Comment (Vibaswan) [reply do.py]: done

File: do.py [hunk 3]
```diff
@@ -459,6 +460,42 @@ def getstatusoutput(command):
...
+    if clean_env.lower() == "true":
+        print("\nCleaning up exsisting env")
+        run(["rm -rf .env"])
+
+    if not os.path.exists(".env"):
+        setup()
+    else:
+        print("virtualenv already exists will be the exsisting one\n")
+
+    py.path = os.path.join(".env", "bin", "python")
```
Comment (ashutshkumr) [inline do.py:477]: Print that now we'll be using following interpreter path.

**Review by ashutshkumr — CHANGES_REQUESTED** [review]
Added all review comments

**Review by ashutshkumr — CHANGES_REQUESTED** [review]
Also, for Dockerfile, why are we only generating python SDK ?

**Review by ashutshkumr — APPROVED** [review]

**Pull request merged**